allennlp/modules/elmo.py

```diff
@@ -121,11 +121,16 @@ def forward(self,    # pylint: disable=arguments-differ
 
     @classmethod
     def from_params(cls, params: Params) -> 'Elmo':
+        # Add files to archive
+        params.add_file_to_archive('options_file')
+        params.add_file_to_archive('weight_file')
```

Comment:
We shouldn't add the weight file to the archive here, I think.  Instead the ELMo code should be able to load itself without the weight file, if `params.loading_from_archive` (because the weight file will be redundant with the model weights, and is just taking up space).  But that's a bigger change that probably belongs in a different PR, so this is fine for now.